camus-kafka-coders/src/main/java/com/linkedin/camus/etl/kafka/coders/JsonStringMessageDecoder.java

```diff
@@ -62,7 +63,12 @@ public void init(Properties props, String topicName) {
     String payloadString;
     JsonObject jsonObject;
 
-    payloadString = new String(payload);
+    try {
+      payloadString = new String(payload, "UTF-8");
+    } catch (UnsupportedEncodingException e) {
+      log.error("Unable to load UTF-8 encoding, falling back to system default", e);
```

Comment:
UTF-8 is a required encoding according to the the JVM specification. Therefore, IMHO this much ceremony is not needed.